### PR TITLE
Register zenithguy.is-a.dev

### PR DESCRIPTION
**Pull request by ZenithGuy1**
Register zenithguy.is-a.dev with TXT record pointing to By Zenith..

### CODE REVIEW

File: domains/zenith.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Zenith's main domain",
+    "owner": {
+        "username": "ZenithGuy1",
+        "email": "matheusbarcelosas@gmail.com"
+    },
+    "record": {
+        "TXT": "By Zenith"
```
Comment (21Z) [inline domains/zenith.json:8]: TXT records alone does nothing to your website.

File: domains/zenithguy.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ZenithGuy1",
+           "email": "matheusbarcelosas@gmail.com",
+           "discord": "1078031080780742707"
+        },
+    
+        "record": {
+            "TXT": "by zenith."
```
Comment (21Z) [inline domains/zenithguy.json:9]: TXT records alone does nothing to your website.

File: domains/cdn.zenith.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Zenith's CDN images subdomain",
+    "owner": {
+        "username": "ZenithGuy1",
+        "email": "matheusbarcelosas@gmail.com"
+    },
+    "record": {
+        "TXT": "By Zenith"
```
Comment (21Z) [inline domains/cdn.zenith.json:8]: TXT records alone does nothing to your website.

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/zenith.json (hunk 1 above)
Comment (ZenithGuy1) [reply domains/zenith.json:8]: I know it does nothing. I dont want it to do anything as well.

**Comment by orxngc** [comment]
Closing PR as silly; you can't have a domain with TXT records alone, we need a complete and accessible site.

**Pull request closed without merging**